src/assets/images/EmptyBoxImage.tsx

```diff
@@ -0,0 +1,38 @@
+export const EmptyBoxImage = (): JSX.Element => {
```

Comment:
we should have a props here that we spread over the `svg` - this allows consumers to style the `svg` if needed